src/app/docs/description-list/description-list-docs.component.html

```diff
@@ -61,11 +61,205 @@
+            captionType="success"
+            imageSource="~/assets/img/guidelines/description-list/description-list-usage-1.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline>
+            <p>Use description list with unordered lists when multiple values are associated with the same terms.
+            </p>
+            <sky-docs-design-guideline-thumbnail
+            caption="Do use unordered lists within description lists for easier scanning of items."
+            captionType="success"
+            imageSource="~/assets/img/guidelines/description-list/description-list-usage-2.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline>
+            <p>Use description list in the "long description" mode when users need to view a set of terms with long text descriptions.
+            </p>
+            <sky-docs-design-guideline-thumbnail
+            caption="Do use 'long description' lists for definitions, glossaries, or other content where the description data are complete sentences or longer."
+            captionType="success"
+            imageSource="~/assets/img/guidelines/description-list/description-list-usage-3.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger">
+            <p>Don't use description list when the information is important and should be emphasized in the information hierarchy. Use <a href="https://developer.blackbaud.com/skyux/components/key-info">key info</a> or other text styles instead.
+            </p>
+            <sky-docs-design-guideline-thumbnail
+            caption="Be cautious using description lists for information that should be called out higher in the information hierarchy."
+            captionType="warning"
+            imageSource="~/assets/img/guidelines/description-list/description-list-usage-4.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+            <sky-docs-design-guideline-thumbnail
+            caption="Do highlight important information with key info."
+            captionType="success"
+            imageSource="~/assets/img/guidelines/description-list/description-list-usage-5.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline>
+            <p>Don't use description list when information can be understood on its own within its context. Use standalone text without a term or label.
+            </p>
+            <sky-docs-design-guideline-thumbnail
+            caption="Be cautious using description lists when information can be understood in context."
```

Comment:
Not a huge deal, but would you mind adding a tab to all these attributes? They should look like so...
```
<sky-docs-design-guideline-thumbnail
    caption="..."
    captionType="..."
    imageSource="..."
>
```

Reply:
Done